.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "build": {
+        "dockerfile": "dockerfile",
+        "context": ".."
+    },
```

Comment:
And this would become `"image" : "github.com/uliegecsm/hwloc-xml-parser:latest"`.

Reply:
Done. In two steps 😄